src/game/client/ev_hldm.cpp

```diff
@@ -502,7 +504,10 @@ void EV_FireGlock1(event_args_t *args)
 		V_PunchAxis(0, -2.0);
 	}
 
-	EV_GetDefaultShellInfo(args, origin, velocity, ShellVelocity, ShellOrigin, forward, right, up, 20, -12, 4);
+	if (cl_righthand.GetFloat() > 0)
```

Comment:
I think it's better to negate Z velocity in `EV_GetDefaultShellInfo` instead of every place that it is called at

Reply:
Yeah it could be formatted better